src/locales/messages.ts

```diff
@@ -314,6 +314,11 @@ export default defineMessages({
     description: 'Cost',
     id: 'Cost',
   },
+  CostExplorerTitle: {
+    defaultMessage: 'EN Cost Explorer',
+    description: 'Cost Explorer',
+    id: 'CostExplorerTitle',
+  },
```

Comment:
We already have `ExplorerTitle`. 

FYI, the component is named explorer, so I skip the "Cost" prefix.

Reply:
make sense just missed that